lib/services/new_answer_service.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:project_eureka_flutter/models/answer_model.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart' as DotEnv;
+import 'dart:async';
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+
+class NewAnswerService {
+  Future<http.Response> postNewAnswer(AnswerModel answer) async {
+    await DotEnv.load();
+
+    final response = await http.post(
+      Uri.http(
+        DotEnv.env['HOST'] + ':' + DotEnv.env['PORT'],
+        '/v1/answers',
+      ),
+      headers: <String, String>{
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+      body: jsonEncode(
+        answer,
```

Comment:
I'm actually surprised that you didn't need to call `.toJson` here, but it seems to be working...

Reply:
i'll leave to json function, just in case we need it in the future